docs/designers-developers/designers/menu-item.md

```diff
@@ -0,0 +1,151 @@
+# MenuItem
+
+MenuItem is a component which renders a button intended to be used in combination with the DropdownMenu Component.
+
+![An image of a MenuItem being highlighted inside of a DropdownMenu component](https://wordpress.org/gutenberg/files/2018/11/MenuItem.png)
+
+1. MenuItem
+
+
+
+## Table of contents
+
+
+
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
+
+
+
+##Design guidelines
+
+
+
+###Usage
+
+The MenuItem button should only be used in combination with the DropdownMenu component.
```

Comment:
I'm not certain this is true. I think the `MenuItem` should be used in combination with `MenuGroup` and it can also go inside `Dropdown` which is a less opinionated `DropdownMenu`. Actually, I think it can't be used with `DropdownMenu` at the moment.

Reply:
Ahhh okay that is helpful. I'll update and to make this more accurate.

Reply:
I just wanted to mention that a `MenuItem` should really only be used with a `MenuGroup`. The reason behind that is their accessibility roles work together (`MenuGroup` by default outputs a role of 'menu', and `MenuItem` outputs a role of 'menuitem'. The `Dropdown` component doesn't output a role afaik).

A `MenuGroup` containing `MenuItem`s can be used within a `Dropdown`. A `MenuGroup` can also have other `MenuGroup`s within it so that menus can be nested.